shell/platform/windows/testing/win32_window_test.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <windowsx.h>
+
+#include "flutter/fml/macros.h"
+#include "flutter/shell/platform/windows/win32_window.h"
+
+namespace flutter {
+namespace testing {
+
+/// Test class for the Win32Window base class. Used to access protected methods
+/// for testing.
+class Win32WindowTest : public Win32Window {
+ public:
+  Win32WindowTest();
+  ~Win32WindowTest();
+
+  // Wrapper for GetCurrentDPI() which is a protected method.
+  UINT GetDpi();
+
+  // |Win32Window|
+  void OnDpiScale(unsigned int dpi) override;
+
+  // |Win32Window|
+  void OnResize(unsigned int width, unsigned int height) override;
+
+  // |Win32Window|
+  void OnPointerMove(double x, double y) override;
+
+  // |Win32Window|
+  void OnPointerDown(double x, double y, UINT button) override;
+
+  // |Win32Window|
+  void OnPointerUp(double x, double y, UINT button) override;
+
+  // |Win32Window|
+  void OnPointerLeave() override;
+
+  // |Win32Window|
+  void OnChar(char32_t code_point) override;
+
+  // |Win32Window|
+  void OnKey(int key, int scancode, int action, int mods) override;
+
+  // |Win32Window|
+  void OnScroll(double delta_x, double delta_y) override;
+
+  // |Win32Window|
+  void OnFontChange() override;
+
+ private:
+  FML_DISALLOW_COPY_AND_ASSIGN(Win32WindowTest);
```

Comment:
Sorry, missed this before. We don't use FML in the desktop embedding since they aren't part of the public API surface. Also, this macro violates current Google style guide (it's mirroring an old, since-deprecated macro that used to be recommended). Please remove the include of fml, and use direct deletion to avoid copying as is done elsewhere in the desktop embeddings ([exampe](https://github.com/flutter/engine/blob/d452dd5c36d7285769bdffc99f9a37976961b3cf/shell/platform/common/cpp/client_wrapper/include/flutter/method_result.h#L21); note that it should be up with the constructor/destructor, not private as with the macro).

Reply:
Done, changed in `win32_flutter_window_test.h` as well